doc/lib_term.md

```diff
@@ -23,7 +23,7 @@ Here is an example clock display from the examples folder, using
 the char_circle and char_line primitives to draw the clock face
 and the hands:
 
-![example image showing a clock](example_lib_term.png)
+![example image showing a clock](lib_term_illustration.png)
```

Comment:
It is quite common on website to put images in dedicated folder, so I'd assume this way they are close enough to each other.